go.mod

```diff
@@ -16,24 +16,33 @@ require (
 	github.com/gorilla/handlers v1.5.1
 	github.com/gorilla/mux v1.8.0
 	github.com/hashicorp/golang-lru v0.5.4
-	github.com/mitchellh/mapstructure v1.1.2
+	github.com/mitchellh/mapstructure v1.5.0
 	github.com/ncw/swift v1.0.47
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.0.2
-	github.com/prometheus/client_golang v1.12.1 // indirect; updated to latest
+	github.com/pkg/errors v0.9.1
```

Comment:
From what I can see the only reason why we are introducing this module is we wrap errors in `tracing` pacakge and we only do it thrice, which makes me feel like we don't need to introduce this dependency and could simply use the standard library wrapping i.e. `fmt.Errorf("foo bar: %w", err)`